app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class AppointmentsController < ApplicationController
+    def index
+        @appointments = Appointment.all
+    end
+
+
+    def new
+        @appointment = Appointment.new
+    end
+
+    def create
+        @appointment = Appointment.new(appointment_params)
+        if @appointment.save
+          redirect_to appointments_path, notice: "Appointment was successfully created."
+        else
+          render :new, alert: "There was an error creating the appointment."
+        end
+    end
+    
+    def appointment_params
+        params.require(:appointment).permit(:scheduled_date, :scheduled_start, :scheduled_end, :note)
```

Comment:
You'll need to add the rest of the relevant columns to the .permit part of params to allow a fully-filled out appointment to be created. The UI currently doesn't allow you to add any more params than specified here, but it's best practice to allow everything on the backend ahead of time so someone else working on the front end (or yourself later on) doesn't forget the change is necessary.

Reply:
Thanks for letting me know, I've added the rest of the database columns to appointment_params and just pushed the change.